### DISCUSSION TITLE
Videos for the workshop

### DISCUSSION THREAD

**Discussion by Ayo-faks** [Azure-Samples/miyagi#107] [Q&A] [answered]
Please are there any video recordings for the workshop?

**Comment by thegovind** [comment, marked as answer]
Thanks for reaching out @Ayo-faks - We plan to have more recordings in the coming months as some of the capabilities that are part of the Copilot stack and still in private preview. Meanwhile, you can checkout - https://www.youtube.com/watch?v=V8dlEvXdGEM&t=144s and https://www.linkedin.com/posts/azure-diy_ai-infused-intelligent-apps-with-microsoft-activity-7077402193665953792-Ijvp
